Update 0061-REWP-pos_rewards.md

Vegaklaus opened the pull request:
- Ersatzvalidators have non-whaling rule
- Upper bound on competition factor
- 2 new AC for parameter changes

Vegaklaus committed:
Update 0061-REWP-pos_rewards.md

davidsiska-vega commented on protocol/0061-REWP-pos_rewards.md:
```diff
@@ -121,5 +116,8 @@ i.e., there is no anti-whaling function applied here (the penalties are removed)
+1. Change of network parameter `reward.staking.delegation.competitionLevel` will change the level of competition of the validators (influences how much stake is be needed for all validators to reach optimal revenue) at the end of the next epoch. Default value 3.1. Minimum value 1 (inclusive). (<a name="0061-REWP-008" href="#0061-REWP-008">0061-REWP-008</a>) 
 1. Change of network parameter `reward.staking.delegation.minimumValidatorStake` will change minimum amount required of own stake a validator has. Minimum stake applies to all validators. it’s referred to as a prerequisite to being considered a validator. Validators not met with the minimum stake will not be all thrown, and in fact unless there’s someone who can replace them no one will be kicked ...
+1. Change of the network parameter 'reward.staking.delegation.optstakemultipolier' is changed to 0 (the reward curve is flat for a validator that exceeds optimal stake), to 0.5 (the reward curve goes down), and 0.1 (the reward curve goes down slightly).(<a name="0061-REWP-010" href="#0061-REWP-010">0061-REWP-010</a>) 
+1. Chanfe of network parsmeter 'reward.staking.delegation.delegatorShare' to 0 (no reward for delegators, 0.99, and 0.5. The share for delegators at the end of the epochs changes accordingly.  (<a name="0061-REWP-011" href="#0061-REWP-011">0061-REWP-011</a>) 
```
```suggestion
1. Change of network parameter 'reward.staking.delegation.delegatorShare' to 0 (no reward for delegators, 0.99, and 0.5. The share for delegators at the end of the epochs changes accordingly.  (<a name="0061-REWP-011" href="#0061-REWP-011">0061-REWP-011</a>) 
```

davidsiska-vega commented on protocol/0061-REWP-pos_rewards.md:
```diff
@@ -121,5 +116,8 @@ i.e., there is no anti-whaling function applied here (the penalties are removed)
 
 ## Change of network parameters
-1. Change of network parameter `reward.staking.delegation.competitionLevel` will change the level of competition of the validators (influences how much stake is be needed for all validators to reach optimal revenue) at the end of the next epoch. Default value 3.1. Minimum value 1 (inclusive). No maximum. (<a name="0061-REWP-008" href="#0061-REWP-008">0061-REWP-008</a>) 
+1. Change of network parameter `reward.staking.delegation.competitionLevel` will change the level of competition of the validators (influences how much stake is be needed for all validators to reach optimal revenue) at the end of the next epoch. Default value 3.1. Minimum value 1 (inclusive). (<a name="0061-REWP-008" href="#0061-REWP-008">0061-REWP-008</a>) 
```
Is the minimum here consistent in what's in `defaults.go` @Vegaklaus ?

gordsport committed:
Update protocol/0061-REWP-pos_rewards.md

Co-authored-by: David Siska <62546419+davidsiska-vega@users.noreply.github.com>

Vegaklaus committed:
Update 0061-REWP-pos_rewards.md

Deleted all parameter range specifications, as there's now a central file for those.
